static/app/views/starfish/components/samplesTable/spanSamplesTable.tsx

```diff
@@ -67,6 +68,7 @@ type Props = {
   avg: number;
   data: SpanTableRow[];
   isLoading: boolean;
+  moduleName: string;
```

Comment:
We have a moduleName type here, that we should probably use instead of `string`
https://github.com/getsentry/sentry/blob/master/static/app/views/starfish/types.tsx#L13
Although its missing a bunch of modules, so we could extend that.

Reply:
+1 to this! If you add the missing modules to `ModuleName` and use `ModuleName` instead of strings that'd be great 👍🏻